packages/components/src/components/assets/CoinLogo/CoinLogo.tsx

```diff
@@ -1,18 +1,19 @@
-import { ImgHTMLAttributes } from 'react';
+import { ImgHTMLAttributes, useState } from 'react';
 import { ReactSVG } from 'react-svg';
 import styled from 'styled-components';
 import { COINS } from './coins';
 
 export type CoinType = keyof typeof COINS;
 
 export interface CoinLogoProps extends ImgHTMLAttributes<HTMLImageElement> {
-    symbol: CoinType;
+    symbol?: CoinType;
+    iconUrl?: string;
     className?: string;
     size?: number;
     index?: number;
 }
 
-const SvgWrapper = styled.div<Omit<CoinLogoProps, 'symbol'>>`
+const Wrapper = styled.div<Omit<CoinLogoProps, 'symbol'>>`
```

Comment:
I am afraid this would create some react-errors. The custom props in the styled-component has to be prefixed with `$`

See: https://styled-components.com/docs/api#transient-props

We have some tooling for this `packages/components/src/utils/transientProps.ts`

Reply:
Didn't create any react errors, but we can add `$` to custom props for sure!